### PR TITLE
Nov 4 Tap Dashboard

### PR DESCRIPTION
**Pull request by stevenelleman**
Screenshot 2024-11-05 at 8 36 28 AM
Screenshot 2024-11-05 at 8 36 37 AM
Screenshot 2024-11-05 at 8 36 43 AM

### CODE REVIEW

File: apps/frontend/src/components/cards/CommunityCard.tsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ export enum DisplayedDashboard {
   STRAVA = "strava",
   GITHUB = "github",
   LANNA_TOTAL_WORKOUTS = "lanna-total-workouts",
+  WEEKLY_TAPS_NOV_4 = "weekly-taps-nov-4",
+
```
Comment (AndrewCLu) [inline apps/frontend/src/components/cards/CommunityCard.tsx:14]: empty space

File: apps/frontend/src/lib/chip/leaderboard.ts [hunk 2]
```diff
@@ -135,7 +135,7 @@ export async function updateTapLeaderboardEntry(
     const updateWeekTapsRequest: UpdateLeaderboardEntryRequest = {
       authToken: session.authTokenValue,
       chipIssuer,
-      entryType: LeaderboardEntryType.WEEK_OCT_27_TAP_COUNT,
+      entryType: LeaderboardEntryType.WEEK_NOV_$_TAP_COUNT,
```
Comment (AndrewCLu) [inline apps/frontend/src/lib/chip/leaderboard.ts:138]: oops

File: apps/frontend/src/pages/community/index.tsx [hunk 3]
```diff
@@ -140,6 +147,14 @@ export default function CommunityPage() {
           communityIssuer,
           LeaderboardEntryType.LANNA_TOTAL_WORKOUT_COUNT
         );
+        weekNov4TapDetails = await getUserLeaderboardDetails(
+          communityIssuer,
+          LeaderboardEntryType.WEEK_NOV_$_TAP_COUNT
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/community/index.tsx:152]: oops

File: apps/frontend/src/pages/community/index.tsx [hunk 4]
```diff
@@ -140,6 +147,14 @@ export default function CommunityPage() {
...
+        );
+        weekNov4TapEntries = await getTopLeaderboardEntries(
+          communityIssuer,
+          LeaderboardEntryType.WEEK_NOV_$_TAP_COUNT
```
Comment (AndrewCLu) [inline apps/frontend/src/pages/community/index.tsx:156]: oops

File: packages/types/src/chip/index.ts [hunk 5]
```diff
@@ -110,6 +110,7 @@ export enum LeaderboardEntryType {
   TOTAL_TAP_COUNT = "TOTAL_TAP_COUNT",
   WEEK_OCT_20_TAP_COUNT = "WEEK_OCT_20_TAP_COUNT",
   WEEK_OCT_27_TAP_COUNT = "WEEK_OCT_27_TAP_COUNT",
+  WEEK_NOV_$_TAP_COUNT = "WEEK_NOV_$_TAP_COUNT",
```
Comment (AndrewCLu) [inline packages/types/src/chip/index.ts:113]: oops

**Review by AndrewCLu — COMMENTED** [review]
You also need to update the computation of new leaderboard entries upon tap, i.e. https://github.com/cursive-team/connections/blob/9f346d96dc4ad14c1977cbb23bd79301f80861ce/apps/frontend/src/lib/chip/leaderboard.ts#L105

File: packages/types/src/chip/index.ts (hunk 5 above)
Comment (stevenelleman) [reply packages/types/src/chip/index.ts:113]: whoops bad replace all my b

**Pull request merged**